src/utils.ts

```diff
@@ -252,7 +251,13 @@ const utils = {
    */
   checkForStream: (obj: {file?: {data: unknown}}): boolean => {
     if (obj.file && obj.file.data) {
-      return obj.file.data instanceof EventEmitter;
+      return (
```

Comment:
Can streams of be represented in non-node environments?

Reply:
Yeah, looks like there is something https://developer.mozilla.org/en-US/docs/Web/API/FileReader#events